### PR TITLE
Fix page description

### PR DESCRIPTION
**Pull request by samuko-things**
**This pull request makes the following changes:**
* Fixes issue #79
* Fixes issue #63

**Checklist:**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me
- [x] I certify that I ran my checklist.

Ping @Ifycode/git-github-training-resource

### CODE REVIEW

**Comment by samuko-things** [comment]
@Ifycode, I've just fixed issues 79 and 63 and this is the pull request, I'd like you to review it.

File: doc/part-1-github-ui/workflows-and-best-practices.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
-{% hint style="success" %}
-PART 1: GITHUB UI
-{% endhint %}
+# ⭐ Workflows and best practices
 
 ## Learning videos
 
 The learning videos in here will talk about workflows and best practices when working with the Github UI. What you will learn will also help you to tackle the tasks that have been provided for you to do after watching the learning videos, and is a base for what you will learn later in part 2.
 
+You will learn the following from the learning videos on this page:
```
Comment (Ifycode) [inline doc/part-1-github-ui/workflows-and-best-practices.md]: Neatly done @samuko-things 👍🏽
But change this line to "You will learn the following in this section:"

**Review by Ifycode — CHANGES_REQUESTED** [review]

**Comment by Ifycode** [comment]
@samuko-things I'm waiting for your changes so that I can move this repo.

**Comment by samuko-things** [comment]
i have made the 

> @samuko-things I'm waiting for your changes so that I can move this repo.

I have made the change pls check it out.

File: doc/part-1-github-ui/workflows-and-best-practices.md [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
-PART 1: GITHUB UI
-{% endhint %}
+# ⭐ Workflows and best practices
 
 ## Learning videos
 
 The learning videos in here will talk about workflows and best practices when working with the Github UI. What you will learn will also help you to tackle the tasks that have been provided for you to do after watching the learning videos, and is a base for what you will learn later in part 2.
 
+You will learn the following in this section:
+
```
Comment (Ifycode) [inline doc/part-1-github-ui/workflows-and-best-practices.md:12]: Approved 🎉

**Review by Ifycode — APPROVED** [review]

**Pull request merged**